js/register.js

```diff
@@ -0,0 +1,54 @@
+function Register() {
+  this.domElement = null;
+    // create an el for this instance
+    // append it to the provided container
+    // set initial attributes (just total)
+  this.initialize = function(container) {
+    console.log(container)
+    this.domElement = $('<div class="register"></div>');
+    container.append(this.domElement);
+    this.total = "";
+  };
+}
+
+Register.prototype.render = function() {
+  var templateString = '<div class="ticket">' +
+		'<h1>Thank You!</h1>' +
+		'<table>' +
+			'<tbody class="entries">' +
+			'</tbody>' +
+			'<tfoot>' +
+				'<tr>' +
+					'<th>Total</th>' +
+					'<th class="total">$0.00</th>' +
+				'</tr>' +
+			'</tfoot>' +
+		'</table>' +
+	'</div>' +
+	'<form class="entry">' +
+		'<input class="newEntry" autofocus placeholder="How Much?">' +
+	'</form>';
+
+  // insert content into `el`
+  // add any event listeners
+
+
+  this.domElement.html(templateString);
+  // this.domElement.find(".ticket").text("");
+
+
+};
+
+Register.prototype.update = function() {
+  event.preventDefault();
```

Comment:
`event.preventDefault();` will only stop the page from refreshing when it's attached to a particular element. If you added something in the render function like this then it would stop the refreshing.
```js
// in prototype.render
var form = this.domElement.find(".entry")
form.on("submit", this.update.bind(this));
```